### DISCUSSION TITLE
Info: Traffic Routing changes (Sept11, 2022)

### DISCUSSION THREAD

**Discussion by darianbjohnson** [Phambili-Tech/Newt_Display#36] [Announcements]
All - I will be launching v 1.1.0-RC1 later this evening. The last step in the change is to update the backend API (which has been enhanced to support additional weather details and a new "riddles" API). This change is effective NOW (Sun, 11 Sep 2022 19:20 GMT;
Sept 11, 3:30 PM Eastern Time).

As always, let me know if you see any craziness.

**Comment by andypiper** [comment]
Worked out ok - I switched my Newt off and on, and at first it didn't connect (at least, I assume it did not - the time was off and the weather/quotation did not show). I did it a second time about 20 minutes later, and it came back up as expected.

**Reply by darianbjohnson** [reply]
Hmm... Interesting. This happens a lot. The first "init" call gets data via an https get call... not sure why it sometimes doesn't connect.... something to look into...
